apps/sms/js/thread_ui.js

```diff
@@ -2531,10 +2531,10 @@ var ThreadUI = {
       // eg. type "a", then delete it.
       // Always remove the the existing results.
       this.toggleRecipientSuggestions();
-      return;
+      return Promise.resolve(null);
```

Comment:
Looks like here you actually changed program flow a bit for the case when everything was deleted from recipients input, earlier it was like this: ```toFieldInput -> searchContact -> toggleRecipientSuggestions```, now it's like this: ```toFieldInput -> searchContact -> toggleRecipientSuggestions -> listContacts```. Probably it doesn't have any consequences yet, but why don't you want use ```Promise.reject()``` here - sounds like a reasonable use case for reject here?